src/components/user_bids_page/UserBidCard.js

```diff
@@ -4,11 +4,12 @@ import './UserBidCard.scss';
 import BidCard from '../bid_card/BidCard';
 
 const UserBidCard = ({ item, bids }) => {
+    const lastThreeBids = bids.length > 3 ? bids.slice(bids.length - 3) : bids;
```

Comment:
To make this more flexible, I think it would be nice to extract the number 3 into a constant, and rename this to `mostRecentBids` - that way it's a little easier to show a different number of recent bids (fewer places to change `3`)